src/app/pages/common/ix-forms/components/ix-file-upload/ix-file-upload.component.scss

```diff
@@ -0,0 +1,35 @@
+:host {
+  color: var(--fg1);
+  display: block;
+
+  margin-bottom: 12px;
+  margin-top: 18px;
+  padding: 8px 0;
+}
+
+.label-container {
+  margin-bottom: 1px;
+  margin-left: 1px;
+
+  .label {
+    font-size: x-small;
+  }
+
+  .tooltip {
+    margin-left: 8px;
+    vertical-align: -4px;
+  }
+}
+
+.input-container {
+  font-size: 12px;
+  position: relative;
+
+  &.disabled {
+    opacity: 0.38;
+  }
+
+  .upload-button {
+    margin-top: 8px;
+  }
+}
```

Comment:
I haven't tested it yet, but can you make sure that keyboard focus states are properly supported?